requirements.txt

```diff
@@ -1,3 +1,6 @@
 # Project dependencies. Please keep this list in alphabetical order.
 
 flask==2.2.2
+python-dotenv==0.21.0
+Flask-WTF==1.0.1
+WTForms==3.0.1
```

Comment:
Let's use lowercased names of dependencies:
```
flask-wtf==1.0.1
wtforms==3.0.1
```
It will help us to use some tools for sorting, because non all tools are case insensitive.

Btw, let's sort these dependencies aphabetically